package/install_osx.sh

```diff
@@ -4,6 +4,6 @@ brew update >/dev/null 2>&1  # This produces a lot of output that's not very int
 
 # Install Python 3.6
 #brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/f2a764ef944b1080be64bd88dca9a1d80130c558/Formula/python.rb
-brew upgrade pyenv 
+brew upgrade pyenv || true
```

Comment:
Can you please clarify this? Is `pyenv` misbehaving somehow, by returning non-successful exit statuses when it should return success? Something else? I tend to feel that `|| true` in shell scripts are a lot like `try/catch: pass` code -- something you don't really want, that is bound to hide away future unexpected failures... Thoughts? :)

Reply:
brew will throw an error if pyenv is already at the latest version.

Reply:
(preliminary note: I stand corrected, the eventual misbehaviour I was referring to was `brew`'s, of course!)

Are you sure? This is not what I am observing on my system (admittedly running a somewhat oldish macOS version, 10.12.6):
```
$ brew list | grep pyenv
pyenv
$ brew update
Already up-to-date.
$ brew upgrade pyenv
(big warning about older macOS version)

Warning: pyenv 1.2.15 already installed
$ echo $?
0
$
```

I somehow insist on this because I'd prefer CI to fail as early as possible, and going with `|| true` means that any transitory but real `brew` failure at this step will be ignored and, most likely, lead to job execution failure at a later point. :)

Reply:
@tmontes does your note at the top mean brew doesn't return 0 when the package is installed?